P/PETSc/build_tarballs.jl

```diff
@@ -341,10 +408,13 @@ platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p
 platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv7l" && libc(p) == "glibc"), platforms)
 platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "x86_64" && libc(p) == "musl"), platforms)
 platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "i686"), platforms)
+platforms = filter(p -> !(p["mpi"] == "openmpi" && Sys.isfreebsd(p)),  platforms)
```

Comment:
I suggest to add a comment here (and for MPICH below) that this excludes only `aarch64-unknown-freebsd`, and that these exclusions can go away once OpenMPI and MPICH have been built for these platforms.